docs/migration/migration8.md

```diff
@@ -1,16 +1,17 @@
 # Migrating from v7 to v8
 
-## Major changes and new features
+## New features
 
-None
+- When using `FormFileGraphType` with type-first schemas, you may specify the allowed media
+  types for the file by using the new `[MediaType]` attribute on the argument or input object field.
 
 ## Breaking changes
 
 - The validation rules' signatures have changed slightly due to the underlying changes to the
-  GraphQL.NET library. Please see the GraphQL.NET v8 migration document for more information.
-- The obsolete (v6 and prior) authorization validation rule has been removed. See the v7 migration
+  GraphQL.NET library.  Please see the GraphQL.NET v8 migration document for more information.
+- The obsolete (v6 and prior) authorization validation rule has been removed.  See the v7 migration
```

Comment:
```suggestion
- The obsolete (v6 and prior) authorization validation rule has been removed. See the v7 migration
```

Reply:
This repository has "properly" two spaces after every period consistently throughout all documentation.